src/language-html/utils.js

```diff
@@ -376,7 +376,10 @@ function inferScriptParser(node) {
       return "markdown";
     }
 
-    if (node.attrMap.type === "application/ld+json") {
+    if (
+      /^([a-z]+-)?importmap$/.test(node.attrMap.type) ||
+      /^application\/([a-z]+\+)?json$/.test(node.attrMap.type)
```

Comment:
I think better use list of possible types than using regexp, just my opinion

Reply:
The reason I didn't go with a list of types is so that it can support arbitrary "application/__foo__+json" and "__foo__-importmap" types. It's on a rarely-used path, so shouldn't cause significant performance issues.

Reply:
I was curious whether substr would be faster. On my machine the regexp is faster in Firefox, but substr is faster in Chrome. https://jsperf.com/mime-matching

Reply:
The fever should also take into account hyphens since they’re used by e.g. GitHub.

Reply:
@j-f1 Where would the hyphens be?

Reply:
Here’s [an example](https://developer.github.com/v3/reactions/) in the wild: `application/vnd.github.squirrel-girl-preview+json`